templates/page.tpl.php

```diff
@@ -115,14 +112,14 @@
 
     <?php if ($messages): ?>
     <section id="messages">
-      <div class="grid-container<?php print ($no_sidebar || $layout == 'cover-1col' ? ' narrow' : ''); ?>">
+      <div class="grid-container<?php print ($is_narrow ? ' narrow' : ''); ?>">
```

Comment:
`$is_narrrow` should be documented at the top of the file.

Reply:
`$has_sidebar` too, I guess?

Reply:
Yes, please ;-)